branchLayout/impl/build.gradle.kts

```diff
@@ -1,43 +1,17 @@
-//val junit: Unit by extra
-dependencies {
-            testImplementation(rootProject.libs.junit)
-        }
+import com.virtuslab.gitmachete.buildsrc.BuildUtils
 
-//val lombok: Unit by extra
 dependencies {
-            compileOnly(rootProject.libs.lombok)
-            annotationProcessor(rootProject.libs.lombok)
-            testCompileOnly(rootProject.libs.lombok)
-            testAnnotationProcessor(rootProject.libs.lombok)
-        }
+    api(project(":branchLayout:api"))
```

Comment:
Btw, any way in Kotlin to use static identifiers for projects? sth like `api(project(branchLayout.api))`?
Don't bother if they need to be defined explicitly... just curious if maybe that's sth built-in

Reply:
(@MaciejG604 bump)

Reply:
Or lemme just add it to #1004 ;p